src/blocks/temperature.rs

```diff
@@ -101,9 +129,17 @@ impl Block for Temperature {
                 .iter()
                 .max()
                 .block_error("temperature", "failed to get max temperature")?;
+            let min: i64 = *temperatures
+                .iter()
+                .min()
+                .block_error("temperature", "failed to get min temperature")?;
             let avg: i64 = (temperatures.iter().sum::<i64>() as f64 / temperatures.len() as f64).round() as i64;
 
-            self.output = format!("{}° avg, {}° max", avg, max);
+            let values = map!("{avg}" => avg,
```

Comment:
Can you use `{average}` instead of `{avg}`? I'd like to keep away from abbreviations in most cases.

Reply:
Do we want `{minimum}` and `{maximum}` also?